actions/promote/promote.js

```diff
@@ -17,28 +17,64 @@
 
 // eslint-disable-next-line import/no-extraneous-dependencies
 const openwhisk = require('openwhisk');
-const { getAioLogger } = require('../utils');
+const { PROJECT_STATUS } = require('../project');
+const {
+    getAioLogger, updateStatusToStateLib, getStatusFromStateLib, PROMOTE_ACTION
+} = require('../utils');
 
 // This returns the activation ID of the action that it called
-function main(args) {
+async function main(args) {
```

Comment:
Have to make this async so that I can await on the getStatusFrom store function.